includes/core/classes/class-assets.php

```diff
@@ -121,6 +121,15 @@ public function add_global_object(): void {
 	 */
 	public function enqueue_scripts(): void {
 		wp_enqueue_style( 'dashicons' );
+
+		if ( has_block( 'gatherpress/venue' ) ) {
+			wp_enqueue_style(
+				'gatherpress-leaflet',
+				'/wp-content/plugins/gatherpress/includes/core/classes/styles/leaflet.css',
```

Comment:
There's contants to get this path. But rather we add this to the build process which also handles dependencies and version.